app/forms/form.rb

```diff
@@ -1,4 +1,22 @@
 class Form
+  class Option
+    attr_reader :id, :name, :description, :hint
+
+    def initialize(id:, name:, description: nil, hint: nil)
+      @id = id
+      @name = name
+      @description = description
+      @hint = hint
+    end
+
+    def ==(other)
+      id == other.id &&
+        name == other.name &&
+        description == other.description &&
+        hint == other.hint
+    end
```

Comment:
is this just used for the spec? or is there some other use for it?

Reply:
Just used to get the spec to pass. I think it's worth having rather than changing the spec